template.md

```diff
@@ -0,0 +1,25 @@
++++
+showonlyimage = true
+draft = false
+image = "img/(filename)"
+date = "2020-MM-DDT12:00:00-05:00"
+title = "(Statue Name), (Location), (Country)"
+photo_credits = ""
+photo_source_url = "(Article or social media source link)"
+<!--- REMOVE THESE COMMENTS BEFORE COMMITING -->
+<!--- Possible categories: colonizers, confederates, racists, rapists, slavers, war-criminals -->
+categories = ["slavers"]
+<!--- Possible tags: municipal-action, renaming, direct-action, private-action -->
+tags = ["municipal-action"]
+weight = 1
++++
```

Comment:
Should also include example of subjects now, e.g.
`subjects: ["Robert E. Lee"]`